common/src/main/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/OnDeviceCertBasedAuthChallengeHandler.java

```diff
@@ -122,7 +123,14 @@ public void alias(String alias) {
      */
     @Override
     public void cleanUp() {
-        //Nothing needed at the moment.
+        final String methodTag = TAG + ":cleanUp";
+        //For on-device CBA, we need to clear the certificate choice cache here so that
+        // the user will be able to login with multiple accounts with CBA
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            WebView.clearClientCertPreferences(null);
+        } else {
+            Logger.warn(methodTag, "Client Cert Preferences cache not cleared due to SDK version < 21 (LOLLIPOP)");
```

Comment:
Could you also add to this message, the possible effect of cleanup not happening.
(what issues/errors user might see)